app/controllers/groceries_controller.rb

```diff
@@ -81,9 +82,27 @@ def recipes
     render json: JSON.parse(res)
   end
 
+  def set_store
+    @grocery_store = GroceryStore
+      .create_with(grocery_store_params)
+      .find_or_create_by(place_id: params[:grocery_store][:place_id])
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.